canonical/ubuntu.com
# Refresh AI landing and add new form

## Done

- Bumped Vanilla version to 4.1 
- Applied content and design changes to landing page as per [doc](https://docs.google.com/document/d/1HP9YdHF11yVQ9SLl0VXhLezDZREsr1GHIKNi9wT1Gv0/edit) and [mock-up](https://www.figma.com/file/sGeCjFvA1kV9Kmvg2S2VBn/23.10-U.com---AI-bubble?node-id=248%3A1446&mode=dev)
- Built new form in the new style as per [copy doc](https://docs.google.com/document/d/1maC3tCmTAgY1G_0nd0aAzw5ZMNeZqbkyawCRMG5Rgxk/edit) and [mock-up](https://www.figma.com/file/sGeCjFvA1kV9Kmvg2S2VBn/23.10-U.com---AI-bubble?node-id=764%3A9597&mode=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13138.demos.haus/ai
- Check against design and doc
- Make sure all CTAs work as expected
- Fill out the form and make sure it submits to marketo

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5577, https://warthogs.atlassian.net/browse/WD-5547
